connector/auth/jwt_token_validation.go

```diff
@@ -143,7 +157,7 @@ func (jv *JwtTokenValidator) getIdentity(jwtString string) (ClaimsIdentity, erro
 	}
 
 	// TODO: Add options verify_aud and verify_exp
-	token, err := jwt.Parse(jwtString, getKey)
+	token, err := jwt.Parse(jwtString, getKey, jwt.WithLeeway(jv.jwtoptions.Leeway))
```

Comment:
I assume this is okay to set every time since the zero value would be a no-op in this case?

Reply:
Right